src/filecoin/FilecoinEngine.ts

```diff
@@ -189,15 +189,15 @@ export class FilecoinEngine extends CurrencyEngine<
     }
 
     const networkFee = mul(txJson.GasLimit.toString(), txJson.GasPremium) // TODO: Include base fee and burn fee somehow?
-    const totalTxAmount = add(nativeAmount, networkFee)
+    const txNativeAmount = mul(add(nativeAmount, networkFee), '-1')
 
     const edgeTransaction: EdgeTransaction = {
       txid: '',
       date: 0,
       currencyCode,
       blockHeight: 0,
-      nativeAmount: `-${totalTxAmount}`,
-      isSend: nativeAmount.startsWith('-'),
```

Comment:
should just be `true`

Reply:
Why do other engines detect sends based on the sign of the number?